### ISSUE TITLE
`conda env export` does not include channels information

### ISSUE THREAD

**Issue by branning** [conda/conda-env#97]
It seems that if you have installed conda packages from anything but the main channel, then you cannot create an environment from the output of `conda env export` due to the fact that the channel information is not included in the YAML output.

I'm using conda-env 2.2.1 with conda 3.13.0 with python 2.7.9 32-bit on Windows 7 64-bit. Here's a minimal example:

```
set MYENV=channeltest
conda create -n %MYENV% -c branning scandir -y
call activate %MYENV%
conda env export -f environment.yml
call deactivate
conda env remove -n %MYENV% -y
conda env create
```

If you run that, and you can install the Windows 32-bit scandir package from binstar, you'll see this error at the end:

    Error: No packages found in current win-32 channels matching: scandir 1.1 py27_0

But if you [edit environment.yml by adding the channel information](https://gist.github.com/branning/d75fa15e92f9ca7dc08c),

```
python fix_yml.py > environment_fix.yml
conda env create --file environment_fix.yml
```

it works as expected, or at least this is how I believe it should work.

**Comment by malev** [comment]
There is more conversation about this in #103
Also we are considering expanding the way `export` works to include from what channel was `X` package installed. Still a thinking about this

**Comment by msarahan** [comment]
closed by #103
